### PR TITLE
Pcp 971 add fraud net to all payments

### LINKED ISSUE #1040: Add FraudNet to all payments. (971)
### Describe the new feature

Currently FraudNet is integrated on PUI, we could make it available to all payment via enable/disable setting.

[Integrate FraudNet into your web app documentation](https://developer.paypal.com/limited-release/fraudnet/integrate/)

### PR DESCRIPTION
**Pull request by Narek13**
**Issue**: #1040

---

### Description

Currently FraudNet is integrated on PUI only.
The PR will add availability to enable FraudNet via setting for all the other payment methods.

### Steps to Test

1. The "FraudNet" enable/disable setting is added in Connection Tab settings.
2. The "FraudNet" script is added into appropriate frontend pages.

---

Closes #1040 .

### CODE REVIEW

File: modules/ppcp-api-client/src/Endpoint/OrderEndpoint.php [hunk 1]
```diff
@@ -94,6 +95,20 @@ class OrderEndpoint {
...
 
+	/**
+	 * True if FraudNet support is enabled in settings, otherwise false.
+	 *
+	 * @var bool
+	 */
+	protected $is_fraudnet_enabled;
+
+	/**
+	 * The FrauNet entity.
```
Comment (Dinamiko) [inline modules/ppcp-api-client/src/Endpoint/OrderEndpoint.php]: Missing `d` in Fraudnet.

File: modules/ppcp-api-client/src/Endpoint/OrderEndpoint.php [hunk 2]
```diff
@@ -120,6 +135,8 @@ class OrderEndpoint {
 	 * @param ApplicationContextRepository $application_context_repository The application context repository.
 	 * @param PayPalRequestIdRepository    $paypal_request_id_repository The paypal request id repository.
 	 * @param SubscriptionHelper           $subscription_helper The subscription helper.
+	 * @param bool                         $is_fraudnet_enabled true if FraudNet support is enabled in settings, otherwise false.
+	 * @param FraudNet                     $fraudnet The FrauNet entity.
```
Comment (Dinamiko) [inline modules/ppcp-api-client/src/Endpoint/OrderEndpoint.php]: Missing `d` in Fraudnet description.

**Review by Dinamiko — APPROVED** [review]

File: modules/ppcp-api-client/src/Endpoint/OrderEndpoint.php (hunk 2 above)
Comment (Narek13) [reply modules/ppcp-api-client/src/Endpoint/OrderEndpoint.php]: fixed: 2506b39c32fc3d3b75f40ba40461bfd4576dee1e

File: modules/ppcp-api-client/src/Endpoint/OrderEndpoint.php (hunk 1 above)
Comment (Narek13) [reply modules/ppcp-api-client/src/Endpoint/OrderEndpoint.php]: fixed: 2506b39c32fc3d3b75f40ba40461bfd4576dee1e

**Pull request merged**